### ISSUE TITLE
Bot attempting to claim characters when ignore_cooldown is false? Is there a way to not autoroll with cooldown?

### ISSUE THREAD

**Issue by joshoea** [chiyadev/MudaeFarm#156]
I have auto claim on which i assume automatically claims, however I have ignore_cooldown as false, which means do not attempt to claim when cooldown is still there right? However, recently the bot just  tried to claim a character in wishlist when cooldown was there already. Is this  a bug or am I doing this wrong. Also, is there a way to not autoroll when cooldown is there so nobody else can claim a wish during cooldown? Just downloaded this today. Kudos to all the developers btw.

**Comment by phosphene47** [comment]
Does this occur repeatedly? It is completely possible for the bot to attempt to claim if it doesn't know about cooldowns. After the bot attempts to claim and Mudae responds with "you're on cooldown," it will begin to respect the cooldown time.

**Comment by joshoea** [comment]
Yes this still happens with ignore_cooldown on false. Maybe after I claim someone I need to click another character, so that way Mudae responds cooldown.

**Comment by phosphene47** [comment]
I think the kakera related part is due to https://github.com/chiyadev/MudaeFarm/issues/128 which I will fix in the next release.

I'm not sure about the character claim cooldown being ignored. Sounds like a possible bug, but I need more information.
